### PR TITLE
bump posthog-js

### PR DESCRIPTION
**Pull request by benarent**
We are seeing data being sent to the postgres database but not posthog, wanted to test this on staging.

### CODE REVIEW

**Comment by avatus** [comment]
This doesn't need to be approved as you're just using the development branch yes?

**Comment by benarent** [comment]
> This doesn't need to be approved as you're just using the development branch yes?

Correct, I'm just trying to debug why I'm not getting any Posthog data.

**Comment by benarent** [comment]
Version 1.43.0

image

**Comment by benarent** [comment]
Bumped to 1.88.1 and still works against our 1.43.0 Cluster. 
image

File: components/Feedback/Feedback.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export default function PageWithJSbasedForm(props) {
 
   const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
     event.preventDefault();
+    console.log(feedback);
```
Comment (avatus) [inline components/Feedback/Feedback.tsx:43]: remove console.log

**Review by avatus — COMMENTED** [review]
Generally, we don't include submodule updates in PRs. Is there a particular reason they are included here? Otherwise the bump looks fine

**Comment by benarent** [comment]
>  Generally, we don't include submodule updates in PRs. Is there a particular reason they are included here? 

For some reason, my setup auto-updates it on each yarn install.

**Review by avatus — APPROVED** [review]

**Pull request merged**